examples/nemo/scripts/KGOs/arch-linux_spack.fcm

```diff
@@ -0,0 +1,19 @@
+%PSYCLONE_HOME       ${PSYCLONE_HOME}
+%NCDF_INC            -I${NCDF_F_HOME}/include -I${NCDF_C_HOME}/include -I${HDF5_HOME}/include
+%NCDF_LIB            -L${NCDF_F_HOME}/lib -lnetcdff -L${NCDF_C_HOME}/lib -lnetcdf
+
+%CPP	             cpp -Dkey_nosignedzero
```

Comment:
They also have it in the repo "arch/arch-linux_gfortran.fcm" and many other fcm files. Which I don't understand because `add_key "key_nosignedzero"` bu I got some issues without having both.